view.rb

```diff
@@ -1,74 +1,94 @@
 module Dialog
-	### Asks if you have an account already.
-	def self.new_or_old_user
-		print "\nHello! Do you have a pre-existing account with us? ('yes' or 'no')\n"
+	def self.new_or_old_user # Asks if you have an account already.
+		print "\nHello! Do you have a pre-existing account with us? ('yes' or 'no'): "
 		gets.chomp.downcase
 	end
 
-	### Initial greeting (first thing user should see straight after entering ruby <file> into the command line)
-	def self.greeting  
-		puts "\nWelcome to RubyBank, where all of your financial dreams come true."
-		puts "\nWould you like to make an account? ('yes' or 'no')" 
+	def self.greeting # Initial greeting (first thing user should see straight after entering ruby <file> into the command line)
+		print "\nWelcome to RubyBank, where all of your financial dreams come true."
+		print "\n\nWould you like to make an account? ('yes' or 'no'): " 
 		gets.chomp.downcase
 	end
 
-	### Says goodbye and mourns the loss of user.
-	def self.goodbye 
-		puts "\nThat is too bad. We are a great bank. Suit yourself! Have a nice day!\n"
-	end
-	### Explains that method is not yet available, check back later.
-	def self.goodbye2			
-		puts "\nThis is not yet a current working method in RubyBank. Check back in a few hours.\nGoodbye!\n\n"
+	def self.goodbye # Says goodbye and mourns the loss of user.
+		print "\nThat is too bad. We are a great bank. Suit yourself! Have a nice day!\n\n"
 	end
 
 	def self.goodbye_cust # The goodbye message for a customer. 
-		puts "\nHave a great day! We will look forward to your return!\n"
+		print "\nHave a great day! We will look forward to your return!\n\n"
 	end
 
 	def self.account_name # this should be the dialog around initializing your bank account. You give your account name.	
-		puts "\nplease enter your the name you wish to associate with the account."
+		print "\nPlease enter your the name you wish to associate with the account: "
 		gets.chomp.downcase
 	end
 
-	def self.account_balance # this should be the dialog around initializing your bank account. You give your account password.	
-		puts "\nplease enter the initial balance you would like to deposit into your new account."
-		gets.chomp.downcase.to_i
-	end
+  def self.signin_name
+    print "\nPlease enter your account name: "
+    gets.chomp.downcase
```

Comment:
Oh, cool. Yeah, I think there are a lot of things that could be 
improved. Should we start brainstorming about our next steps? I think 
the first order of business is to make a manager login/signup an option. 
That shouldn't be too hard if you explain to me how you would go about 
creating a manager in the database. The other way I thought to get 
around the capitalization problem was to do an if statement that 
excludes all but the accepted entry parameters or one that includes only 
the accepted parameters.

Any ideas for next steps on this project? According to my curriculum, 
there should be a rails project coming up this week. I'll ask Benny what 
he thinks.

On 08/14/2015 12:10 PM, em77 wrote:
>
> In view.rb 
> <https://github.com/erikasland/RubyBank/pull/6#discussion_r37096878>:
>
> >  		gets.chomp.downcase
> >  	end
> >
> > -	def self.account_balance # this should be the dialog around initializing your bank account. You give your account password.	
> > -		puts "\nplease enter the initial balance you would like to deposit into your new account."
> > -		gets.chomp.downcase.to_i
> > -	end
> > +  def self.signin_name
> > +    print "\nPlease enter your account name: "
> > +    gets.chomp.downcase
>
> I was saying this is the same thing I was talking about earlier in 
> view file regarding downcase. I think there's a better way to handle 
> searching for a name and making and accomplishing what your original 
> goal was, which was minimizing the possibility for errors based on a 
> differences in capitalization. We can figure that out in next round of 
> changes. I have some ideas.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/erikasland/RubyBank/pull/6/files#r37096878>.
>